test/test_fuzz.py

```diff
@@ -1257,6 +1281,34 @@ def _check_frameToRangeEquivalence(self, test, expect):
         lambda self, t=tst, e=exp: TestFramesToFrameRange._check_frameToRangeEquivalence(self, t, e))
 
 
+class TestFrameSetFromRangeConstructor(unittest.TestCase):
+    """
+    Exercise the TestFrame.from_range() constructor. Due to the sheer number of permutations, we'll add most tests dynamically.
+    """
+
+    def _check_fromRange(self, start, end, step, expect):
+        """
+        Harness to test if the FrameSet.fromRange call works properly.
+        :param expect: the string to use to build the expected FrameSet, which will be normalized for comparison
+        :param start: the start frame
+        :param end: the end frame
+        :return: None
+        """
+        e = FrameSet(expect)
+        r = FrameSet.from_range(start, end, step)
+        m = u'FrameSet.fromRange({0}, {1}) != {2!r}: got {3!r}'
+        self.assertEqual(r, e, m.format(start, end, e, r))
+        m = u'FrameSet.fromRange({0}, {1}) returns {2}: got {3}'
+        self.assertIsInstance(r, FrameSet, m.format(start, end, FrameSet, type(r)))
+
+
+# add tests dynamically
+for name, start, end, step_, exp in FRAME_SET_FROM_RANGE_SHOULD_SUCCEED:
+    setattr(
+        TestFrameSet, 'testFromRange%s' % name,
+        lambda self, s=start, e=end, step=step_, exp=exp: TestFrameSetFromRangeConstructor._check_fromRange(self, s, e, step, exp))
```

Comment:
Copy-paste error, should be `TestFrameSetFromRangeConstructor` instead of `TestFrameSet`? This may have been why the py2 CI test failed.

Reply:
My bad, my Python 2.7 tests were not properly setups.